### DISCUSSION TITLE
PyAnnote error

### DISCUSSION THREAD

**Discussion by CXhk87** [kaixxx/noScribe#38] [Q&A]
Hi !
Thanks for NoScribe. I am new and do not succed to have my audio transcripted: and I do not understand why. This is the message I get. Can someone help me please ?
![Capture d’écran 2023-11-30 à 22 48 18](https://github.com/kaixxx/noScribe/assets/152559385/51c310f0-e2e5-466b-a57f-cc689774e2a5)

**Comment by kaixxx** [comment]
Hi
First of all: can you tell me your macOS version? This would help to understand the error better.

There seems to be a problem with the hardware acceleration MPS, so you have to switch it off. In order to do that, you must close noScribe and locate the file "config.yml" in ~user/noScribe. Open the file in a text editor and change the line "pyannote_xpu: mps" to "pyannote_xpu: cpu". Save the file, restart noScribe and try again. It will be slower but hopefully working.
